### PR TITLE
feat: Plug building of an image into the task manager

### PR DESCRIPTION
**Pull request by benoitf**
### What does this PR do?
Allow to go back to a build of OCI image through the task manager

depends on
 - [ ] https://github.com/containers/podman-desktop/pull/1724

### Screenshot/screencast of this PR

https://user-images.githubusercontent.com/436777/225675346-c277d671-780a-49c3-baa3-5acb69e28ff9.mp4

### What issues does this PR fix or reference?

first step of https://github.com/containers/podman-desktop/issues/1343

### How to test this PR?

Needs to apply first https://github.com/containers/podman-desktop/pull/1724

then, check that if you start to build an image, you see the task in the task manager and you can go back to it.

Change-Id: If60f491744b200f65e267c22559739ec313a7edf

### CODE REVIEW

**Review by lstocchi — COMMENTED** [review]
Tested, works perfectly. It's really nice, GJ 👏 
Waiting to approve it until the other PR is merged, in case of some changes there.

Should the `do not disturb` toggle do something? 
image

**Comment by benoitf** [comment]
@lstocchi for now it does nothing, discussion is happening on https://github.com/containers/podman-desktop/pull/1724

**Comment by benoitf** [comment]
> Waiting to approve it until the other PR is merged, in case of some changes there.

@lstocchi it has been rebased (no change in the code for this PR) on top of the main branch (where PR of task manager has been merged)

**Review by lstocchi — APPROVED** [review]

**Comment by benoitf** [comment]
I'll merge by the end of the day if no objection

**Review by cdrage — APPROVED** [review]
Tested and code LGTM. 

We should as part of the UI, make sure that when things are "sent" to the notification bar, there is a little indicator that it's running? (like it turns purple, or there's a little number beside it showing how many tasks are running?)

**Comment by benoitf** [comment]
yes we should improve the task manager. Only first step is there.

**Comment by benoitf** [comment]
the 'enhancement' issue https://github.com/containers/podman-desktop/issues/1744

**Pull request merged**
